R/addin.R

```diff
@@ -16,7 +16,8 @@
 #' @param Rmd_file path to a .Rmd file. If `NULL`, use the active document.
 #'
 #' @export
-confl_create_post_from_Rmd <- function(Rmd_file = NULL) {
+confl_create_post_from_Rmd <- function(Rmd_file = NULL, shiny = TRUE,
```

Comment:
Could you change `shiny = TRUE` to `interactive = interactive()`? Then it can be automatically non-Shiny when executed on console, and I think `interactive` is more general term.

Reply:
I think that's a great idea! I made the change.